src/modules/create-market/components/create-market-form/create-market-form.styles.less

```diff
@@ -175,10 +175,10 @@
         content: "";
         display: none;
         height: 0.875rem;
-        left: 0.37rem;
+        left: 6px;
```

Comment:
Why pixels over rem here?

Reply:
Because of a persistent, annoying bug. When the positioning of the inner circles of the radio buttons was set using rems instead of px, they would randomly be a little off center (sometimes, but not always, and not consistently across elements - go figure). Using px seems to have fixed it (or at least the randomness didn't occur while I was looking).